firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -732,6 +733,81 @@ extension TelemetryWrapper {
             case halfView = "half-view"
             case fullView = "full-view"
         }
+
+        // Awesomebar
+        public enum UrlbarTelemetry: String {
+            case nChars = "n_chars"
+            case nResults = "n_results"
+            case nWords = "n_words"
+            case selectedResult = "selected_result"
+            case selectedResultSubtype = "selected_result_subtype"
+            case engagementType = "engagement_type"
+            case provider
+            case reason
+            enum Reason: String {
+                case pause
+            }
+
+            case sap
+            enum Sap: String {
+                case urlbar
+                case urlbarNewtab = "urlbar_newtab"
+            }
+
+            case interaction
+            enum Interaction: String {
+                case typed
+                case pasted
+                case returned
+                case restarted
+                case refined
+                case persistedSearchTerms = "persisted_search_terms"
+                case persistedSearchTermsRestarted = "persisted_search_terms_restarted"
+                case persistedSearchTermsRefined = "persisted_search_terms_refined"
+            }
```

Comment:
might be valuable to add some comments for future developers to understand what the different types are. Unless we have the documentation in Glean, then can we reference that dictionary?

Reply:
I added the links to the docs which have more details about the event and the keys